FlowCal/io.py

```diff
@@ -1760,6 +1760,38 @@ def __array_finalize__(self, obj):
         if hasattr(obj, '_resolution'):
             self._resolution = copy.deepcopy(obj._resolution)
 
+    # override __reduce__ and __setstate__ to fix Issue #277
+    # Adapted from https://stackoverflow.com/a/26599346
```

Comment:
Remove these. I would rather have this explained in a docstring in each function with more detail. See below.